openpdf/src/main/java/com/lowagie/text/pdf/FontSelector.java

```diff
@@ -67,18 +67,40 @@
 
     protected ArrayList<Font> fonts = new ArrayList<>();
 
+    public FontSelector() {
+        FontFactory.register("font-fallback/LiberationSans-Regular.ttf", "sans");
+        Font font = FontFactory.getFont("sans", BaseFont.IDENTITY_H);
+        fonts.add(font);
+    }
+
+    /**
+     * change the color of default font in <CODE>FontSelector</CODE>.
+     * @param color the <CODE>Color</CODE> of default font
+     */
+    public void setDefaultColor(Color color) {
+        fonts.get(fonts.size() - 1).setColor(color);
+    }
+
+    /**
+     * change the size of default font in <CODE>FontSelector</CODE>.
+     * @param size the size of default font
+     */
+    public void setDefaultSize(float size) {
+        fonts.get(fonts.size() - 1).setSize(size);
+    }
+
     /**
      * Adds a <CODE>Font</CODE> to be searched for valid characters.
      * @param font the <CODE>Font</CODE>
      */
     public void addFont(Font font) {
         if (font.getBaseFont() != null) {
-            fonts.add(font);
+            fonts.add(fonts.size() - 1, font);
```

Comment:
I think, these are equivalent:

- `fonts.add(fonts.size() - 1, font)`
- `fonts.add(font)`

An ArrayList will .add() per default to the end of the list.

Reply:
they are no equivalent actually.
Test code:
![image](https://user-images.githubusercontent.com/60342704/119480193-fee94d00-bd83-11eb-84ee-4089517217cb.png)
output:
![image](https://user-images.githubusercontent.com/60342704/119480242-0c063c00-bd84-11eb-8231-85b181eb3a84.png)